backend/models/dtos/stats_dto.py

```diff
@@ -141,3 +141,19 @@ def __init__(self):
     # avg_completion_time = IntType(serialized_name='averageCompletionTime')
     organisations = ListType(ModelType(OrganizationListStatsDTO))
     campaigns = ListType(ModelType(CampaignStatsDTO))
+
+
+class TaskStatsDTO(Model):
+    """ DTO for tasks stats for a single day """
+
+    date = DateType(required=True)
+    mapped = IntType(serialized_name="mapped")
+    validated = IntType(serialized_name="validated")
+    invalidated = IntType(serialized_name="invalidated")
+    bad_imagery = IntType(serialized_name="badImagery")
+
+
+class AllTaskStatsDTO(Model):
```

Comment:
Let's have this class as TaskStatsDTO - and the individual date DTO named as `TaskStats' - some way to know that these two are related and one of this is the building block for another.

For example we have: 
* ProjectComment, ProjectCommentsDTO
* ProjectContribDTO, ProjectContribsDTO
* UserContribution, ProjectContributionsDTO